common/djangoapps/embargo/forms.py

```diff
@@ -11,15 +11,11 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 
-from embargo.models import (
-    EmbargoedCourse, EmbargoedState, IPFilter,
-    RestrictedCourse
-)
-from embargo.fixtures.country_codes import COUNTRY_CODES
+from embargo.models import IPFilter, RestrictedCourse
 
 
-class CourseKeyValidationForm(forms.ModelForm):
```

Comment:
I think we're also using this form for course modes admin, so I would be really hesitant to remove it entirely.